doc/ref/gappkg.xml

```diff
@@ -173,5 +187,1995 @@ users.
+
+</Subsection>
+
+</Section>
+
+
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="Having an InfoClass">
+<Heading>Having an InfoClass</Heading>
+
+<!-- TODO: (CW) the Example package does not declare an InfoExample, 
+     and it ought to do so! It might be a good idea to suggest 
+     _where_ to put this declaration.-->
+
+<Index Key="InfoClass" Subkey="for a GAP package"><C>InfoClass</C></Index>
+It is a good idea to declare an <C>InfoClass</C> for your package. This  gives
+the package user the opportunity  to  control  the  verbosity  of  output
+and/or the possibility of receiving debugging information
+(see&nbsp;<Ref Sect="Info Functions"/>).  Below,  we  give  a  quick
+overview of its utility.
+<P/>
+An <C>InfoClass</C> is defined with  a  <C>DeclareInfoClass(  <A>InfoPkgname</A>  );</C>
+statement and may be set to have an initial <C>InfoLevel</C>  other  than  the
+zero default (which means no <C>Info</C> statement is to  output  information)
+via a <C>SetInfoLevel( <A>InfoPkgname</A>, <A>level</A>  );</C>  statement.  An  initial
+<C>InfoLevel</C> of 1 is typical.
+<P/>
+<C>Info</C> statements have the form: 
+<C>Info( <A>InfoPkgname</A>, <A>level</A>, <A>expr1</A>, <A>expr2</A>, ...);</C>  
+where  the  expression  list  <C><A>expr1</A>,  <A>expr2</A>,  ...</C>
+appears just like it would in a <C>Print</C> statement. The only difference is
+that the expression list is  only  printed  (or  even  executed)  if  the
+<C>InfoLevel</C> of <A>InfoPkgname</A> is at least <A>level</A>.
+
+</Section>
+
+
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="The Banner">
+<Heading>The Banner</Heading>
+
+<!-- TODO: say that the defaul banner is constructed from PackageInfo.g -->
+
+<Index Subkey="for a GAP package">banner</Index>
+
+When the package is loaded, &GAP; will display a default package banner, 
+constructed from the package metadata provided in the <F>PackageInfo.g</F> file.
+<P/>
+Alternatively, the package may establish its own banner by assigning a string
+to the <C>BannerString</C> field of the record argument of <C>SetPackageInfo</C>
+in the <F>PackageInfo.g</F> file.
+<P/>
+If you will be designing a banner for your package, it is a good idea to suggest
+there how to access package  documentation. For example, the banner of the
+<Package>Example</Package> package says:
+<P/>
+<Log><![CDATA[
+For help, type: ?Example package
+]]></Log>
+<P/>
+In order for this to display the introduction of the
+<Package>Example</Package>  package the index-entry
+<C>&tlt;Index&tgt;Example package&tlt;/Index&tgt;</C>
+was added just before the first paragraph of the introductory section  in
+the file <F>doc/example.xml</F> of the <Package>Example</Package> package.
+(it uses <Package>GAPDoc</Package> for documentation -
+you will need some different scheme to achieve this using
+the <F>gapmacro.tex</F> format).
```

Comment:
Remove the last part in parenthesis?

Reply:
Checked and removed some other mentions of gapmacro. Now the only one left is "Some packages still use for their manuals the old <Q>gapmacro</Q> format, support for which may be discontinued in the future. We encourage authors of those packages to eventually convert their documentation &GAPDoc;."